content/en/docs/tutorials/security/apparmor.md

```diff
@@ -16,7 +16,7 @@ permissions to confine programs to a limited set of resources. AppArmor can be c
 application to reduce its potential attack surface and provide greater in-depth defense. It is
 configured through profiles tuned to allow the access needed by a specific program or container,
 such as Linux capabilities, network access, file permissions, etc. Each profile can be run in either
-*enforcing* mode, which blocks access to disallowed resources, or *complain* mode, which only reports
+*enforcing* mode, which blocks access to disallowed resources, or *warn* mode, which only reports
```

Comment:
```suggestion
*enforced* mode, which blocks access to disallowed resources, or *complaining* mode, which only reports
```
As per https://ubuntu.com/server/docs/security-apparmor the only two modes are: Enforced/confined and Complaining/learning. There is no warn mode. @haripriya9647  can you elaborate on your change?